src/styles/style.css

```diff
@@ -248,7 +248,11 @@ input[type=text]::placeholder {
 }
 
 .invalid-feedback {
-	font-size: 1.05rem;
+	font-size: 1.15rem;
+	font-family: monospace;
+	white-space: pre-wrap;
+	padding-left: 50px;
+	text-indent: -50px;
```

Comment:
I'm sorry I'm not super well-versed in the tradeoffs, but I noticed that a lot of the rest of the sizes in this file are defined in `rem` rather than `px` ... should these be in `rem` as well?

Reply:
Good point! This is ta rick to get a "hanging indent", which isn't critical at all, but I think its nice for these multiline messages.